Adding configuration example for retain modes

ZuluWhiskey opened the pull request:
Reading this page of the documentation on its own didn't help me understand how to configure my Frigate instance. But when I found https://github.com/blakeblackshear/frigate/discussions/2447, I was able to understand how to add it to my configuration. I've added the example given in that discussion to help future readers of the page.

I myself am still not 100% confident with the configuration setup - If possible it may be worth someone helping to also add the `all` option into the configuration example if that's possible? eg. `all` retained for 1 day, `motion` for 7 and `active_objects` for 14?
Would it worth adding object retention examples here too? eg.
```
record:
  events:
    retain:
      default: 14
      mode: all
      objects:
        person: 30
        car: 14
```
And my PR is for the global configuration, I can't seem to find an example of camera specific retention unfortunately so haven't added that.

ZuluWhiskey committed:
Adding configuration example for retain modes

Reading the documentation on its own didn't help me but when I found https://github.com/blakeblackshear/frigate/discussions/2447 I was able to understand how to add this to my configuration. I've added the example given in that discussion to help future readers of the page.

NickM-27 commented on docs/docs/configuration/record.md:
```diff
@@ -42,3 +42,16 @@ The same options are available with events. Let's consider a scenario where you
 - With the `motion` option all segments for the duration of the event with motion would be saved. This means any segment where a car drove by in the street, person walked by, lighting changed, etc. would be saved. 
 - With the `active_objects` it would only keep segments where the object was active. In this case the only segments that would be saved would be the ones where the car was driving up, you going inside, you coming outside, and the car driving away. Essentially reducing the 4 hours to a minute or two of event footage. 
+
+A configration example of the above retain modes where `motion` is stored for 7 days and `active objects` are store for 14 would be as follows:
```
I think this is a good change but could be reworded slightly to make things more clear.

"above retain modes where `motion` is stored for 7 days" -> "above retain modes where all `motion` segments are stored for 7 days"

"and `active objects` are store for 14" -> "and `active objects` events are stored for 14 days"

NickM-27 replied:
Thanks looks good. I think it would be better to put the object specific retention separately so as to not confuse. Probably would fit under its own `## Object Specific Recording` header since it doesn't necessarily fit with the other topic

ZuluWhiskey replied:
What about `### Object Specific Retention` instead? And therefore be a part of the `## What do the different retain modes mean?` header, rather than it's own separate topic?

NickM-27 replied:
Looks good to me

ZuluWhiskey committed:
Update record.md

Added suggested changes and have also added wording beneath the example mentioning the configuration can be added on a per camera basis.
Have also built on the example to add object specific retentions timings - Not sure if it would be preferred to have it all within one example to not complicate things?

Let me know your thoughts

ZuluWhiskey committed:
Update record.md

Created Object Specific Retention header

blakeblackshear committed:
Typo
